orchestrator/services/order_service.py

```diff
@@ -21,6 +21,7 @@
 from orchestrator.order_status import OrderStatus
 from orchestrator.services.wallet_service import WalletService
 from orchestrator.transaction_history import TransactionHistory
+from utility.services.crypto_to_fiat import get_cogs_amount
```

Comment:
Orchestrator service has no role in the rate conversion. We should not update any file in this directory

Reply:
Removed completely from Orchestrator service and added event handler in serverless.yml of Utility service